test/extensions/filters/http/grpc_http1_bridge/http1_bridge_filter_test.cc

```diff
@@ -86,11 +86,11 @@ TEST_F(GrpcHttp1BridgeFilterTest, StatsHttp2HeaderOnlyResponse) {
 
   Http::TestResponseHeaderMapImpl response_headers{{":status", "200"}, {"grpc-status", "1"}};
   EXPECT_EQ(Http::FilterHeadersStatus::Continue, filter_.encodeHeaders(response_headers, true));
-  EXPECT_EQ(1UL, decoder_callbacks_.clusterInfo()
+  EXPECT_EQ(0UL, decoder_callbacks_.clusterInfo()
```

Comment:
Please make another copy of the test, which disables the new runtime feature and verifies the old counter values